config/geoserver/abraid/styles/abraid_raster.sld

```diff
@@ -15,6 +15,7 @@
                             <sld:ColorMapEntry color="#ffffcb" opacity="1" quantity="0.5"/>
                             <sld:ColorMapEntry color="#cf93ba" opacity="1" quantity="0.75"/>
                             <sld:ColorMapEntry color="#a44883" opacity="1" quantity="1.0"/>
+                            <sld:ColorMapEntry color="#eaeaea" opacity="1" quantity="+9999"/>
```

Comment:
Obviously the + is not strictly necessary but I presume you're using it to clearly distinguish from -9999?